tests/phpunit/tests/rest-api/rest-pattern-directory-controller.php

```diff
@@ -21,6 +21,15 @@ class WP_REST_Pattern_Directory_Controller_Test extends WP_Test_REST_Controller_
 	 */
 	protected static $contributor_id;
 
+	/**
+	 * An instance of WP_REST_Pattern_Directory_Controller class.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @var WP_REST_Pattern_Directory_Controller
+	 */
+	private static $controller;
```

Comment:
There is no need for this to be a class property here. It is only used in this test.

Reply:
I've refactored this property. Now it's used in several tests.